logstash-core/src/main/java/org/logstash/common/BufferedTokenizerExt.java

```diff
@@ -20,17 +20,16 @@
 
 package org.logstash.common;
 
-import org.jruby.Ruby;
-import org.jruby.RubyArray;
-import org.jruby.RubyClass;
-import org.jruby.RubyObject;
-import org.jruby.RubyString;
+import org.jruby.*;
```

Comment:
It seems IDE made this change, worth to revert.

Reply:
addressed with 727e3b1b073aa090c8430b6e87a85affb6155f68